client/van_site_config_create.go

```diff
@@ -18,7 +18,18 @@ func (cli *VanClient) VanSiteConfigCreate(ctx context.Context, spec types.VanSit
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "skupper-site",
 		},
-		Data: map[string]string{},
+		Data: map[string]string{
+			"name":                   types.DefaultVanName,
```

Comment:
With skupper-cli, if the site name is not specified explicitly it defaults to the name of the namespace the site is in. I suspect that is not much better than a fixed name in many cases, but I *think* I have a very mild preference for it.